### PR TITLE
Initial PR for base example

### PR DESCRIPTION
**Pull request by sean-m-sullivan**
Initial PR for base example, mainly copied from tower_configuration, but tests have been modified to work pulling data from external.

### CODE REVIEW

File: .github/collections/requirements.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+collections:
+- name: awx.awx
+- name: sean_m_sullivan.tower_configuration
```
Comment (Tompage1994) [inline .github/collections/requirements.yml]: I assume this is a mistake as this should really be redhat_cop.tower_configuration

File: .github/workflow-config/release.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
```
Comment (Tompage1994) [inline .github/workflow-config/release.yml]: Are we likely to be doing any releases from this repo? If not, this shouldn't be here.

File: .github/workflows/galaxy-release.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
```
Comment (Tompage1994) [inline .github/workflows/galaxy-release.yml]: Same with this, should this file be here?

File: base_example/configure_tower.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  hosts: localhost
+  connection: local
+  # Define following vars here, or in tower_configs/tower_auth.yml
+  # tower_hostname: ansible-tower-web-svc-test-project.example.com
+  # tower_username: admin
+  # tower_password: changeme
+
+  collections:
+   - awx.awx
+   - sean_m_sullivan.tower_configuration
```
Comment (Tompage1994) [inline base_example/configure_tower.yml]: Needs changing to redhat_cop.tower_configuration

File: base_example/configure_tower_export_model.yml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  vars:
+    workflow_job_templates_secure_logging: false
+  # Define following vars here, or in tower_configs/tower_auth.yml
+  # tower_hostname: ansible-tower-web-svc-test-project.example.com
+  # tower_username: admin
+  # tower_password: changeme
+
+  collections:
+   - awx.awx
+   - sean_m_sullivan.tower_configuration
```
Comment (Tompage1994) [inline base_example/configure_tower_export_model.yml]: Needs changing to redhat_cop.tower_configuration

File: base_example/tower_configs/tower_settings_individuale.yml [hunk 6]
```diff
@@ -0,0 +1,46 @@
+---
+tower_settings_individuale:
```
Comment (Tompage1994) [inline base_example/tower_configs/tower_settings_individuale.yml]: Is there a reason this reads `individuale` rather than `individual`?

Comment (Tompage1994) [reply base_example/tower_configs/tower_settings_individuale.yml]: As well as the filename, and the usage in the playbook

**Review by Tompage1994 — CHANGES_REQUESTED** [review]
Looks mostly good. Just a few points to cover!

**Comment by aoyawale** [comment]
besides the comments tom mentioned, looks good.

File: .github/collections/requirements.yml (hunk 1 above)
Comment (sean-m-sullivan) [reply .github/collections/requirements.yml]: Until we get a new release this is temporary, the one I have on galaxy incorporates all our changes + my pull requests as of Saturday, so none of the fixed I did yesterday. Mainly its the tower settings change its incorporating.

File: .github/workflow-config/release.yml (hunk 2 above)
Comment (sean-m-sullivan) [reply .github/workflow-config/release.yml]: Good point.

File: .github/workflows/galaxy-release.yml (hunk 3 above)
Comment (sean-m-sullivan) [reply .github/workflows/galaxy-release.yml]: Same good point. I mainly did a copy paste from ours, and started removing things, but didnt remove this.

File: base_example/configure_tower_export_model.yml (hunk 5 above)
Comment (sean-m-sullivan) [reply base_example/configure_tower_export_model.yml]: See above, Temporary.

File: base_example/tower_configs/tower_settings_individuale.yml (hunk 6 above)
Comment (sean-m-sullivan) [reply base_example/tower_configs/tower_settings_individuale.yml]: I think we arent using this, and its this way in our normal tower_configuration, I think autocorrect decided to have a field day.

Comment (sean-m-sullivan) [reply base_example/tower_configs/tower_settings_individuale.yml]: Removed

File: base_example/configure_tower_export_model.yml (hunk 5 above)
Comment (sean-m-sullivan) [reply base_example/configure_tower_export_model.yml]: Removed this file

File: .github/workflows/galaxy-release.yml (hunk 3 above)
Comment (sean-m-sullivan) [reply .github/workflows/galaxy-release.yml]: Removed this file

File: .github/workflow-config/release.yml (hunk 2 above)
Comment (sean-m-sullivan) [reply .github/workflow-config/release.yml]: Removed this file

**Comment by Tompage1994** [comment]
@sean-m-sullivan This can now be changed to redhat_cop

**Review by Tompage1994 — APPROVED** [review]

**Pull request merged**